frontend/mr-admin-flate/src/components/filter/Avtalefilter.tsx

```diff
@@ -27,7 +28,7 @@ export function Avtalefilter() {
 
   return (
     <>
-      <div className={styles.filter_container}>
+      <div className={classNames(styles.filter_container)}>
```

Comment:
Trenger vi `classNames` her når et bare er ett classname?

Reply:
Nei, godt spottet. Det var opprinnelig to klasser. Jeg fjerner!